docs/content/reference/migration/migration-0-23.md

```diff
@@ -81,3 +81,6 @@ Either:
 
 The former is subject to (double-precision) floating point precision loss (still microsecond precision for the next century), while the latter is lossless.
 
+## 🐍 Python: removed `rr.log_components()`, `rr.connect()`, and `rr.connect_tcp()`
+
+These functions were [deprecated](migration-0-22#python-api-changes) in 0.22 and are no longer available. See the [0.22 migration guide](migration-0-22.md) for detailed information about the new APIs and how to migrate to them.
```

Comment:
The 0.22 migration guide does not mention `rr.connect()` and `rr.connect_tcp()` being deprecated, should we explain it here?

Reply:
I clarified stuff.